src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -243,6 +243,10 @@ def addRuntimeMonitors(self, task):
         monitoring.PerformanceMonitor.maxPSS = maxpss
         monitoring.PerformanceMonitor.softTimeout = softTimeout
         monitoring.PerformanceMonitor.hardTimeout = hardTimeout
+        monitoring.section_("NATSMonitor")
+        monitoring.NATSMonitor.maxPSS = maxpss
```

Comment:
These attributes should actually be something needed for connecting to the server (url, port, etc), just as done above for the DashboardMonitor (not PerformanceMonitor).